.license_template

```diff
@@ -1,2 +1,2 @@
-// Copyright {20\d{2}(-20\d{2})?} IOTA Stiftung{(?:, .+)?}
-// SPDX-License-Identifier: Apache-2.0
+{(\/\/ Copyright.*\n)*?}// {(Modifications )?}Copyright {(\(c\) )?}{20\d{2}(-20\d{2})?} IOTA Stiftung{(, Fondazione Links)?}{(, Filancore GmbH)?}
+{(\/\/ Copyright.*\n)*}// SPDX-License-Identifier: Apache-2.0
```

Comment:
Would this update to the template make sense? ^^

```suggestion
{(\/\/ Copyright.*\n)*?}// {(Modifications )?}Copyright {(\(c\) )?}{20\d{2}(-20\d{2})?} IOTA Stiftung{(?:, .+)?}
// SPDX-License-Identifier: Apache-2.0
```

This would:
- keep the list of non-IOTA contributors open, so that we don't have to extend it for contributions from currently unlisted contributors
- remove the `// Copyright` prefix from the second line, as I didn't find any cases where we needed it ^^;;